src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs

```diff
@@ -34,7 +34,7 @@
 
 namespace Windows.UI.Xaml.Controls
 {
-	public partial class ComboBox : Selector
+	public partial class ComboBox : Selector, IThemeChangeAware
```

Comment:
In [this doc](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/themeresource-internals.md):

> Some non-FrameworkElement types may also need to react to the theme change. This is supported by the IThemeChangeAware internal interface

That seems to imply that this interface is only useful for non-FrameworkElement types, but `ComboBox` is FrameworkElement`

Is the doc outdated? or should things work in a different way?

Reply:
The doc is probably outdated. It used for all framework elements to react to theme changes, if needed.